docs/api/IoT.js-API-Timers.md

```diff
@@ -9,61 +9,84 @@ The following shows timer module APIs available for each platform.
+```javascript
+var timeout = setTimeout(function() {
+  // Do something what will be executed after one second.
+}, 1000);
+```
+
+### clearTimeout(timeout)
+
+* `timeout` {Timeout} A Timeout object as returned by `setTimeout()`.
+
+Cancels a `Timeout` object created by `setTimeout()`.
+
+Example:
+
+```javascript
+var timeout = setTimeout(function() { }, 1000);
+...
+clearTimeout(timeout);
+```
 
-### `close()`
+### setInterval(callback, delay[, args..])
+* `callback` {Function} The function to call when the timer elapses.
+  * `...args` {any}
+* `delay` {number} The number of milliseconds to wait before calling the `callback`.
+* `...args` {any} - Optional arguments to pass when the `callback` is called.
+* Returns: {Timeout}
 
-When called, the active Timeout object will not require the IoT.js event loop to remain active. If there is no other activity keeping the event loop running, the process may exit before the Timeout object's callback is invoked.
+Schedules repeated execution of `callback` every `delay` milliseconds. Returns a `Timeout` for use with `clearInterval()`. If `callback` is not a function, a `TypeError` will be thrown.
 
-## Module Functions
+Example:
 
-### `setTimeout(callback, delay[, args..])`
-* `callback <Function([args..])>`
-* `delay <Number>`
-* `...args <any>` - Optional arguments to pass when the callback is called.
-* Returns: `<Timeout>`
+```javascript
+var timeout = setInterval(function() {
+  // Do something what will be executed repeatadly one time per second.
+}, 1000);
+```
 
-Schedules to call one-time to `callback` function after `delay` milliseconds.
+### clearInterval(timeout)
 
-### `clearTimeout(timeoutObject)`
-*  `timeoutObject <Timeout>`
+* `timeout` {Timeout} A Timeout object as returned by `setInterval()`.
 
-Disables a `Timeout` object from `setTimeout` method.
+Cancels a `Timeout` object created by `setInterval()`.
 
-### `setInterval(callback, delay[, args..])`
-* `callback <Function([args..])>`
-* `delay <Number>`
-* `...args <any>` - Optional arguments to pass when the callback is called.
-* Returns: `<Timeout>`
+Example:
 
-Schedules repeated execution of `callback` every `delay` milliseconds. If callback is not a function, a `TypeError` will be thrown.
+```javascript
+var timeout = setInterval(function() { }, 1000);
+...
+clearInterval(timeout);
+```
+
+## Class: Timeout
+
+This object is created internally and is returned from `setTimeout()` and `setInterval()`.
+
+
+### timeout.activate()
+
+When called, requests that the IoT.js event loop not exit so long as the Timeout is active.
 
-### `clearInterval(intervalObject)`
-* `intervalObject <Timeout>`
+### timeout.close()
 
-Disables a `Timeout` object from `setInterval` method.
+When called, the active `Timeout` object will not require the IoT.js event loop to remain active. If there is no other activity keeping the event loop running, the process may exit before the `Timeout` object's callback is invoked.
```

Comment:
will not force the IoT.js event loop
If there are no other scheduled activites, the process may exit...